scene/resources/particle_process_material.cpp

```diff
@@ -595,14 +610,14 @@ void ParticleProcessMaterial::_update_shader() {
 	if (emission_curve.is_valid()) {
 		code += "	parameters.color.rgb *= 1.0 + texture(emission_curve, vec2(lifetime)).r;\n";
 	}
-	code += "}\n";
+	code += "}\n\n";
 
 	code += "vec3 calculate_initial_position(inout uint alt_seed) {\n";
 	code += "	float pi = 3.14159;\n";
-	code += "	float degree_to_rad = pi / 180.0;\n";
```

Comment:
I removed `degree_to_rad` in some functions where it's not used.

I suppose the idea here was to always make it available in case new code would need it. Happy to bring it back if that's the case.